pypeit/coadd2d.py

```diff
@@ -893,11 +910,22 @@ def compute_offsets(self, offsets):
         Args:
             offsets (:obj:`list` or :obj:`str`):
                 Value that guides the determination of the offsets.
-                It could be a list of offsets, or a string, or None.
+                It could be a list of offsets, or a string.
 
         """
+        msgs.info('Get Offsets')
 
-        pass
+        if (self.objid_bri is None) and (offsets == 'auto'):
+            msgs.error('Offsets cannot be computed because no unique reference object '
+                       'with the highest S/N was found. To continue, provide offsets in `Coadd2DPar`')
+
+        # 1) a list of offsets is provided by the user (no matter if we have a bright object or not)
+        if isinstance(offsets, (list, np.ndarray)):
+            msgs.info('Using user input offsets')
+            # use them
+            self.offsets = self.check_input(offsets, type='offsets')
+            offsets_method = 'user input'
+            self.offsets_report(self.offsets, offsets_method)
 
     def compute_weights(self, weights):
         """
```

Comment:
This is in longer a dummy method that is completely overloaded by the children so please update the docs.